src/components/buttons/ServiceButton.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { colors } from '../../style/color';
+import adjustColorBrightness from '../../utils/adjustColorBrightness';
+
+const ServiceButton = ({
+    onClickButton = () => { console.log("Service button clicked") },
+    width = '160px',
+    height = '160px',
+    borderRadius = '15px',
+    backgroundColor = '#1DB954',
+    imgLink = 'https://uxwing.com/wp-content/themes/uxwing/download/brands-and-social-media/spotify-white-icon.png',
```

Comment:
```suggestion
    imgLink,
```
Il faudrait que imgLink n'est pas d'argument par défaut ou une image qui signifie rien

Reply:
C'est fait